tests/base/account.spec.ts

```diff
@@ -91,20 +87,21 @@ test.describe('Account address book actions', { annotation: {type: 'Account Dash
    */
 
   test('I can add my first address',{ tag: '@address-actions', }, async ({page}, testInfo) => {
-    // If account has no address, Address Book redirects to the 'Add New Address' page.
-    // We expect this to be true before continuing.
-    let addNewAddressTitle = page.getByRole('heading', {level: 1, name: selectors.newAddress.addNewAddressTitle});
-    testInfo.skip(await addNewAddressTitle.isHidden(), `Heading "Add New Addres" is not found, please check if an address has already been added.`);
     const accountPage = new AccountPage(page);
 
+    let addNewAddressTitle = page.getByRole('heading', {level: 1, name: selectors.newAddress.addNewAddressTitle});
+    if(await addNewAddressTitle.isHidden()) {
+      await accountPage.deleteAllAddresses();
+      await page.goto(slugs.account.addressNewSlug);
```

Comment:
Ideally, a testInfo annotation is added here so the report reflects that addresses have been deleted.


```suggestion
      await page.goto(slugs.account.addressNewSlug);
      testInfo.annotations.push({ type: 'Notification: deleted addresses', description: `Addresses were found and deleted before this test was ran.` });
```